### DISCUSSION TITLE
INVALID_CREDENTIALS({'desc': 'Invalid credentials', 'info': '80090308: LdapErr: DSID-0C090569, comment: AcceptSecurityContext error, data 52e, v4563'},)

### DISCUSSION THREAD

**Discussion by ganeshrrhce** [netbox-community/netbox#14100] [Help Wanted!]
INVALID_CREDENTIALS({'desc': 'Invalid credentials', 'info': '80090308: LdapErr: DSID-0C090569, comment: AcceptSecurityContext error, data 52e, v4563'},)

I am getting this error, Please help me

**Comment by BelhanafiAbdelmadjid** [comment]
did you find a solution ?
